test/unit/specs/store/distribution.spec.js

```diff
@@ -106,6 +137,35 @@ describe(`Module: Fee Distribution`, () => {
       })
     })
 
+    describe(`getRewardsFromAllValidators`, () => {
+      it(`success`, async () => {
+        const validators = [
+          { operator_address: `cosmosvaloper1address1` },
+          { operator_address: `cosmosvaloper1address2` },
+        ]
+        await actions.getRewardsFromAllValidators({ state, dispatch }, validators)
+        expect(dispatch).toBeCalledTimes(2)
+        expect(dispatch).toBeCalledWith(
+          `getRewardsFromValidator`,
+          validators[0].operator_address
+        )
+        expect(dispatch).toBeCalledWith(
+          `getRewardsFromValidator`,
+          validators[1].operator_address
+        )
+      })
+
+      it(`fails`, async () => {
+        const validators = [
+          { operator_address: undefined },
+          { operator_address: `cosmosvaloper1address2` },
+        ]
+        dispatch = jest.fn(async () => Promise.reject(Error(`invalid address`)))
```

Comment:
dispatch does not need the jest.fn wrapping, the simplified version should work
```suggestion
        dispatch = () => Promise.reject(Error(`invalid address`))
```

Reply:
I like to test if the dispatches were called with the right parameters, happy to discuss it afterwards tho